frontend/public/static/locale/es/translate.ftl

```diff
@@ -10,6 +10,15 @@ entitydetails-editor-button-save = Guardar
 entitydetails-editor-button-suggest = Sugerir
 
 
+## Entity Navigation
+## Shows next/previous buttons.
+
+entitynavigation-next = <glyph></glyph>Siguiente
+    .title = Go To Next String (Alt + Down)
+entitynavigation-previous = <glyph></glyph>Anterior
+    .title = Go To Previous String (Alt + Up)
```

Comment:
Hi @guerojeff, Any chance you could translate the `.title` attributes to Spanish? :)

Reply:
_Ir a la cadena anterior (Alt + Arriba)_